docs/custom-matchers/writing-custom-matcher.md

```diff
@@ -15,8 +16,9 @@ const endsWithMatcher = pattern => {
       const wrapRegExp = new RegExp(`(.*)\.${pattern}$`); // In production application pattern should definitely be escaped!
 
       return {
-        unwrap: pattern,
-        wrap: action.type.match(wrapRegExp)[1],
+        id: `ends-with${pattern}`,
+        unwrappedType: pattern,
+        wrap: type => `${action.type.match(wrapRegExp)[1]}.${type}`,
```

Comment:
https://github.com/salsita/redux-elm/blob/54630bdbf0638c39af66fe3092432c298d664995/docs/custom-matchers/writing-custom-matcher.md Should be explained here:

> `wrap` is a function which can be used for wrapping the action back